Makefile

```
@@ -21,36 +24,72 @@ DIR_PROBE := "$(CUR_DIR)/cmd/probe"
 build: build-runner build-probe
 
 build-runner: deps-runner
-	@cd ${DIR_RUNNER} && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=${PROBE_VERSION}'" -o "$(CUR_DIR)/bin"
+	@cd $(DIR_RUNNER) && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=$(PROBE_VERSION)'" -o "$(CUR_DIR)/bin"
```

FTR no need to `cd` here; also shouldn't this include the runner bersion instead?
```suggestion
	@go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=$(RUNNER_VERSION)'" -o "$(CUR_DIR)/bin" $(DIR_RUNNER)
```

---

I've addressed the `cd` piece - but the version remains the same for the same reason as above :)